website/docs/r/api_management_logger.html.markdown

```diff
@@ -0,0 +1,101 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_api_management_logger"
+sidebar_current: "docs-azurerm-resource-api-management-logger"
+description: |-
+  Manages an API Management Logger.
+---
+
+# azurerm_api_management_logger
+
+Manages an API Management Logger.
+
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West US"
+}
+
+resource "azurerm_application_insights" "example" {
+  name                = "example-appinsights"
+  location            = "${azurerm_resource_group.example.location}"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  application_type    = "Other"
+}
+
+resource "azurerm_api_management" "example" {
+  name                = "example-apim"
+  location            = "${azurerm_resource_group.example.location}"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  publisher_name      = "My Company"
+  publisher_email     = "company@terraform.io"
+
+  sku {
+    name     = "Developer"
+    capacity = 1
+  }
+}
+
+resource "azurerm_api_management_logger" "example" {
+  name                = "example-logger"
+  api_management_name = "${azurerm_api_management.example.name}"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  description         = "This logger is created by Terraform."
+
+  application_insights {
+    instrumentation_key = "${azurerm_application_insights.example.instrumentation_key}"
+  }
+}
+```
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of this Logger, which must be unique within the API Management Service. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group in which the API Management Service should be exist. Changing this forces a new resource to be created.
```

Comment:
since this is referring to the RG of an existing APIM Service - we can phrase this as:

```suggestion
* `resource_group_name` - (Required) The name of the Resource Group in which the API Management Service exists. Changing this forces a new resource to be created.
```